docs/src/changelog.md

```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 <!-- The line "## <square-bracket>Unreleased</square-bracket>" is replaced by the release/release.py script with the new release version and release date. -->
 
+## [Unreleased]
+
+### Deployment notes
+
+If your currently installed *Quality-time* version is v4.10.0 or older, please read the v5.0.0 deployment notes first.
+
+### Added
+
+- In the popups in the dashboard, show the percentage next to the number of metrics with a specific status. Closes [#7946](https://github.com/ICTU/quality-time/issues/7946).
```

Comment:
"In the popups in the dashboard" sounds wonky, but I'm not sure how to reword. Perhaps "In the dashboard popups" ?

Reply:
Or perhaps it's more clear to talk about "dashboard tooltips"?